test/UserDbSpec.hs

```diff
@@ -19,11 +20,11 @@ import Database.Persist.Types (Filter)
 import Servant
 
 import Api.User
-import Config (App(..), Config(..), Environment(..), makePool)
+import Config (AppT(..), Config(..), Environment(..), makePool)
 import Models
 import qualified Data.Text as T
 
-runAppToIO :: Config -> App a -> IO a
+runAppToIO :: Config -> AppT IO a -> IO a
```

Comment:
We can continue using the synonym `App` for the common case of `AppT IO`.

Reply:
Right. Fixed that